coalib/processes/Processing.py

```diff
@@ -145,30 +150,34 @@ def autoapply_actions(results,
             else:
                 not_processed_results.append(result)
                 continue
+        if action not in bear_actions or action in result.actions:
```

Comment:
This condition ensures that if action is in `bear_actions` then it must be in `result.actions`.